holmes.go

```diff
@@ -152,6 +152,11 @@ func (h *Holmes) startDumpLoop() {
 			continue
 		}
 
+		if h.opts.ReachCPUMax(cpu) {
+			h.logf("[Holmes] current cpu percent [%v] is greater than the cpu max [%v], don't create profile", cpu, h.opts.CPUMaxPercent)
+			return
```

Comment:
`return` should be `continue`, maybe next loop will less than the cpu max.

Reply:
thanks